docs/source/users/getting_started.md

```diff
@@ -0,0 +1,67 @@
+---
+jupytext:
+  formats: md:myst
+  text_representation:
+    extension: .md
+    format_name: myst
+    format_version: 0.13
+kernelspec:
+  display_name: Python 3
+  language: python
+  name: python3
+language_info:
+  codemirror_mode:
+    name: ipython
+    version: 3
+  file_extension: .py
+  mimetype: text/x-python
+  name: python
+  nbconvert_exporter: python
+  pygments_lexer: ipython3
+  version: 3.11.9
+---
+
+# Getting started with `pyrealm`
+
+This page helps you to get started with using `pyrealm` from the ground up.
+
+## Prerequisites
+
+Before installing `pyrealm`, make sure you have python installed. Here are the
+versions for
+
+- [Windows](https://www.python.org/downloads/windows/),
+- [Linux](https://www.python.org/downloads/source/) - there usually is a python
+ version installed natively which should work,
+- and [macOS](https://www.python.org/downloads/macos/) - you can also install
+ python using Homebrew by typing `brew install python3` into the terminal.
+
+The `pyrealm` package can be easily installed using the python `pip` package.
+Mostly, your python version should come with `pip` readily available. If this
+is not the case, the
+[pip documentation](https://pip.pypa.io/en/stable/installation/#get-pip-py)
+shows you how to install `pip`.
+
+## Installing `pyrealm`
+
+When the above prerequisites are fulfilled, you can simply install `pyrealm` by
+typing the command `pip install pyrealm` into the command line.
```

Comment:
I'd add a short section on what `command line` means here: opening a terminal or powershell or windows CMD shell and typing commands in. Maybe mention the common `pip` or `python` not being found issue and point to various OS guides about PATH variables?